doc/source/examples/analysis/custom_parallel_analysis.ipynb

```diff
@@ -71,12 +71,14 @@
   {
```

Comment:
<p>Image seems to be missing? (or maybe ReviewNB doesn't render it properly anymore?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MDAnalysis/UserGuide/pull/241/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>

Reply:
<p>Just pinging this again, there doesn't seem to be any output displayed for jobs.visualize().</p>